.github/workflows/docker-build.yml

```diff
@@ -14,33 +14,74 @@ on:
 
 jobs:
 
-  trigger:
-    runs-on: ubuntu-latest
-    # only do this if we don't have a pull request
-    if: ${{ github.event_name != 'pull_request' }} 
+  # trigger:
+  #   runs-on: ubuntu-latest
+  #   # only do this if we don't have a pull request
+  #   if: ${{ github.event_name != 'pull_request' }} 
+  #   steps:
+  #   - uses: lasith-kg/dispatch-workflow@v1
+  #     id: workflow-dispatch_develop
+  #     name: 'Trigger re-build of desktop docker image "develop"'
+  #     with:
+  #       dispatch-method: workflow_dispatch
+  #       repo: kasm-workspaces-core-images
+  #       owner: LCAS
+  #       ref: refs/heads/develop # or main
+  #       workflow: docker-latest.yml # Or Workflow ID
+  #       token: ${{ secrets.ORGA_GH_TOKEN }} # GitHub Token With Relevant Permissions
+  #   - uses: lasith-kg/dispatch-workflow@v1
+  #     id: workflow-dispatch_nvidia_develop
+  #     name: 'Trigger re-build of desktop docker image "nvidia_develop"'
+  #     with:
+  #       dispatch-method: workflow_dispatch
+  #       repo: kasm-workspaces-core-images
+  #       owner: LCAS
+  #       ref: refs/heads/nvidia_develop # or main
+  #       workflow: docker-latest.yml # Or Workflow ID
+  #       token: ${{ secrets.ORGA_GH_TOKEN }} # GitHub Token With Relevant Permissions
```

Comment:
same as previous - shall we remove this?

Reply:
yep, residual of the work in progress